docs/release/updating-to-osg-36.md

```diff
@@ -17,6 +17,52 @@ pilot jobs and/or issues with data transfer.
 If you have verified that your VO(s) support token-based pilot submission and HTTP-based data transfers,
 use this document to update your OSG software to OSG 3.6.
 
+Updating the OSG Repositories
+-----------------------------
+
+!!! note
+    Before updating the OSG repository, be sure to turn off any OSG services. Consult the sections below
+    that match your situation.
+
+1.  First, clean the yum cache:
+
+        :::console
+        root@host # yum clean all --enablerepo=*
+
+1.  Then, remove the old series Yum repositories:
+
+        :::console
+        root@host # rpm -e osg-release
+
+    This step ensures that any local modifications to `*.repo` files will not prevent installing the new series repos.
+    Any modified `*.repo` files should appear under `/etc/yum.repos.d/` with the `*.rpmsave` extension.
+    After installing the new OSG repositories (the next step) you may want to apply any changes made in the `*.rpmsave`
+    files to the new `*.repo` files.
+
+    !!! note
+        Please disable the ``osg-upcoming`` repository when upgrading to the new release series.
+
+1.  Update your [Yum repositories](../common/yum.md#install-the-osg-repositories) to OSG 3.6
+
+1.  Finally, Update software:
+
+        :::console
+        root@host # yum update
+
+    !!! warning
+        -   Please be aware that running `yum update` may also update other RPMs.
+            You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the
+            `yum` command.
+        -   Watch the yum update carefully for any messages about a `.rpmnew` file being created.
+            That means that a configuration file had been edited, and a new default version was to be installed.
+            In that case, RPM does not overwrite the edited configuration file but instead installs the new version with
+            a `.rpmnew` extension.
+            You will need to merge any edits that have made into the `.rpmnew` file and then move the merged version
+            into place (that is, without the `.rpmnew` extension).
+            Watch especially for `/etc/lcmaps.db`, which every site is expected to edit.
+
```

Comment:
There should be a final step that tells users to proceed to other sections for manual configuration updates that they may need to do, depending on the type of host they're updating.